data-raw/_helper.R

```diff
@@ -0,0 +1,24 @@
+library(here)
+
+internal_data_path <- here("R/sysdata.rda")
+
+save_to_sysdata <- function(obj, obj_name) {
+  # Create a new environment for storing sysdata contents
+  sysdata_env <- new.env(hash = FALSE)
+
```

Comment:
question: delete all these scripts since they do not seem to be replaced by *_dt.R scripts?
```
data-raw/can_prov_cases.R
data-raw/cases_deaths_subset.R
data-raw/counts_subset.R
data-raw/covid_case_death_rates.R
data-raw/covid_incidence_county_subset.R
data-raw/covid_incidence_outliers.R
```

note: originally, `_dt` was meant to be "data table".  But it seems a huge pain to selectively rename stuff to say `_tbl`.

Reply:
RE note: hmm, I will consider renaming.

Reply:
RE question: I think those files are also local to you. I only have a couple non-`_dt` files left (`jhu_confirmed_cumulative_num` and `state_census`)